### PR TITLE
Add installation section to README

### PR DESCRIPTION
**Pull request by captainpete**

### CODE REVIEW

**Comment by captainpete** [comment]
Nice project :)

File: tests/testthat/test-LRU.R [hunk 1]
```diff
@@ -29,8 +29,7 @@ describe('Using LRU cache', {
   test_that('Test peek', {
     cache <- LRUcache(2)
     cache$set('hello', 'world')$set('cache', 'money')
-    expect_is(cache$last_accessed('cache'), 'POSIXct')
-    # did not update the timestamp metadata
+    expect_equal(cache$peek('hello'), 'world')
     expect_true(cache$last_accessed('hello') < cache$last_accessed('cache'))
```
Comment (robertzk) [inline tests/testthat/test-LRU.R]: `s/last_accessed/last_get` in these tests.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
-# cacher [![codecov.io](http://codecov.io/github/kirillseva/cacher/coverage.svg?branch=master)](http://codecov.io/github/kirillseva/cacher?branch=master)[![Build Status](https://travis-ci.org/kirillseva/cacher.svg)](https://travis-ci.org/kirillseva/cacher)
+# cacher
```
Comment (robertzk) [inline README.md]: Note to self: We'll have to remember to remove this from the README after merging upstream.

**Review by robertzk — CHANGES_REQUESTED** [review]
Thanks @captainpete! Looking good with one minor comment. I'll leave a final review to @kirillseva.

**Comment by peterhurford** [comment]
Thanks for the submission! Can you change the title of the PR and add a PR description? It is not very descriptive as to what is happening at the moment.

File: R/lru.R [hunk 3]
```diff
@@ -25,10 +30,15 @@ LRUcache_ <- R6::R6Class("LRUcache_",
         rm(list = name, envir = private$data)
       }
     },
-    last_accessed = function(name) {
+    last_get = function(name) {
       stopifnot(is.character(name) && length(name) == 1)
       stopifnot(name %in% ls(private$data))
-      private$peek(name)
+      private$data[[name]]$last_get
+    },
```
Comment (peterhurford) [inline R/lru.R]: Why not keep both `last_accessed` and `last_get`? Then you avoid making a breaking change.

File: R/lru.R [hunk 4]
```diff
@@ -52,25 +63,21 @@ LRUcache_ <- R6::R6Class("LRUcache_",
       }
       # Check for eviction
       while (private$check_for_eviction(name, size)) { private$evict() }
-      private$data[[name]] <- list(value = value, timestamp = Sys.time())
+      private$data[[name]] <- list(value = value, last_get = time, last_set = time, size = size)
```
Comment (peterhurford) [inline R/lru.R]: I personally don't think the value of renaming `timestamp` is worth the breaking change.

File: R/lru.R [hunk 5]
```diff
@@ -114,7 +121,8 @@ LRUcache_.character <- R6::R6Class("LRUcache_.character", inherit = LRUcache_,
         "B" = 0,
         "K" = 1,
         "M" = 2,
-        "G" = 3)
+        "G" = 3,
+        "T" = 4)
```
Comment (peterhurford) [inline R/lru.R]: You have that much data...? o.O

File: README.md (hunk 2 above)
Comment (peterhurford) [reply README.md]: Could we just ask @captainpete to revert here?

File: README.md [hunk 6]
```diff
@@ -1,4 +1,18 @@
...
+
+***fork warning:*** this fork of [kirillseva/cacher](https://github.com/kirillseva/cacher)
```
Comment (peterhurford) [inline README.md]: ^ Don't want this in the upstream repo either. ;)

File: README.md [hunk 7]
```diff
@@ -33,3 +47,12 @@ cache3$forget("key4")                    # you can also explicitly forget a cert
 cache3$exists("key4")
 [1] FALSE
 ```
+
+Install with [devtools](https://github.com/hadley/devtools).
```
Comment (peterhurford) [inline README.md]: Can you add a header like `## Installation`

**Comment by captainpete** [comment]
Ah! Sorry the other stuff on master wasn't meant to make it into this PR. I've reset the branch, only intended to do the README changes :)

Have added the `## Installation` header as suggested.

File: tests/testthat/test-LRU.R (hunk 1 above)
Comment (captainpete) [reply tests/testthat/test-LRU.R]: Thanks. Something like e28ddaf?

**Comment by peterhurford** [comment]
@captainpete Ah, perfect, thanks! 👍

**Review by peterhurford — APPROVED** [review]

**Comment by peterhurford** [comment]
@captainpete I approve, but I'd want @robertzk to release his rejection before merging.

**Review by robertzk — APPROVED** [review]

**Comment by robertzk** [comment]
@captainpete The other changes looked interesting too, feel free to re-submit!

File: R/lru.R (hunk 3 above)
Comment (captainpete) [reply R/lru.R]: That's a good point, maybe `last_accessed` is a better name than `last_get`, might `s/accessed/get/g`. Will do another PR :)

File: R/lru.R (hunk 4 above)
Comment (captainpete) [reply R/lru.R]: Agreed. Will refactor.

File: R/lru.R (hunk 5 above)
Comment (captainpete) [reply R/lru.R]: 0.064T ;)

**Comment by captainpete** [comment]
Thanks!

**Comment by kirillseva** [comment]
@captainpete thank you for your contribution <3

**Pull request merged**